### PR TITLE
Update workflow names

### PR DESCRIPTION
**Pull request by ericcornelissen**
**Issue:** n/a
**Alexa rank:** n/a
  

### Checklist
  - [n/a] I updated the JSON data in `_data/simple-icons.json`
  - [n/a] I optimized the icon with SVGO or SVGOMG
  - [n/a] The SVG `viewbox` is `0 0 24 24`

### Description

Most importantly gives unique names to `create-release.yml` and `merge-release.yml` workflows. I have gotten confused multiples times looking at [the Actions tab](https://github.com/simple-icons/simple-icons/actions) when I was trying to debug the release automation.

Other than that, this attempts to improve and unify the names of all workflows. Suggestions are welcome :slightly_smiling_face:

### CODE REVIEW

File: .github/workflows/create-release.yml
Comment (mondeja) [inline .github/workflows/create-release.yml:1]: Since the `add-labels` workflow is named using "Pull Request" instead of the acronym, does not make sense use that here also? Additionaly, shouldn't the name be singular ("Pull Request" instead of "Pull Requests" or "PRs")?

```suggestion
name: Create Release Pull Request
```

File: .github/workflows/merge-release.yml
Comment (mondeja) [inline .github/workflows/merge-release.yml:1]: The same here:

```suggestion
name: Merge Release Pull Request
```

File: .github/workflows/create-release.yml
Comment (ericcornelissen) [reply .github/workflows/create-release.yml]: Good suggestion, I agree. Writing it in full I feel like singular makes more sense.

**Review by mondeja — APPROVED** [review]
LGTM! I'll wait for the revision from another maintainer before the merge.

**Review by service-paradis — APPROVED** [review]
LGTM 💯

**Pull request merged**
